src/main/webapp/WEB-INF/web.xml

```diff
@@ -14,106 +14,88 @@
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation=
-    "http://xmlns.jcp.org/xml/ns/javaee
-     http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
-    version="3.1">
-
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xmlns.jcp.org/xml/ns/javaee" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd" version="3.1">
   <listener>
     <listener-class>codeu.controller.ServerStartupListener</listener-class>
   </listener>
-
   <servlet>
     <servlet-name>LoginServlet</servlet-name>
     <servlet-class>codeu.controller.LoginServlet</servlet-class>
   </servlet>
-
   <servlet-mapping>
     <servlet-name>LoginServlet</servlet-name>
     <url-pattern>/login</url-pattern>
   </servlet-mapping>
-
   <servlet>
     <servlet-name>ConversationServlet</servlet-name>
     <servlet-class>codeu.controller.ConversationServlet</servlet-class>
   </servlet>
-
   <servlet-mapping>
     <servlet-name>ConversationServlet</servlet-name>
     <url-pattern>/conversations</url-pattern>
   </servlet-mapping>
-
   <servlet>
     <servlet-name>ChatServlet</servlet-name>
     <servlet-class>codeu.controller.ChatServlet</servlet-class>
   </servlet>
-
   <servlet-mapping>
     <servlet-name>ChatServlet</servlet-name>
     <url-pattern>/chat/*</url-pattern>
   </servlet-mapping>
-  
   <servlet>
     <servlet-name>RegisterServlet</servlet-name>
     <servlet-class>codeu.controller.RegisterServlet</servlet-class>
   </servlet>
-
   <servlet-mapping>
     <servlet-name>RegisterServlet</servlet-name>
     <url-pattern>/register</url-pattern>
   </servlet-mapping>
-  
   <servlet>
     <servlet-name>ProfileServlet</servlet-name>
     <servlet-class>codeu.controller.ProfileServlet</servlet-class>
   </servlet>
-
-
   <servlet-mapping>
     <servlet-name>ProfileServlet</servlet-name>
     <url-pattern>/profile/*</url-pattern>
   </servlet-mapping>
-
   <servlet>
     <servlet-name>ActivityFeedServlet</servlet-name>
     <servlet-class>codeu.controller.ActivityFeedServlet</servlet-class>
   </servlet>
-
   <servlet-mapping>
     <servlet-name>ActivityFeedServlet</servlet-name>
     <url-pattern>/activityfeed</url-pattern>
   </servlet-mapping>
-
   <servlet>
     <servlet-name>AdminServlet</servlet-name>
     <servlet-class>codeu.controller.AdminServlet</servlet-class>
   </servlet>
-
   <servlet-mapping>
     <servlet-name>AdminServlet</servlet-name>
     <url-pattern>/admin</url-pattern>
   </servlet-mapping>
-
   <servlet>
     <servlet-name>ControlPanelServlet</servlet-name>
     <servlet-class>codeu.controller.ControlPanelServlet</servlet-class>
   </servlet>
-
   <servlet-mapping>
     <servlet-name>ControlPanelServlet</servlet-name>
     <url-pattern>/control_panel</url-pattern>
   </servlet-mapping>
-
   <servlet>
     <servlet-name>LogoutServlet</servlet-name>
     <servlet-class>codeu.controller.LogoutServlet</servlet-class>
   </servlet>
-
```

Comment:
I don't think there's a need to remove the whitespace here.

Reply:
I must've run the formatter on this file accidentally! I'll make the changes